src/iter/repeat.rs

```diff
@@ -0,0 +1,108 @@
+use super::*;
+use super::internal::*;
+use std::usize;
+
+pub struct Repeat<T> {
+    element: T,
+}
+
+pub fn repeat<T: Clone>(elt: T) -> Repeat<T> {
+    Repeat{element: elt}
```

Comment:
I'd suggest just running `rustfmt repeat.rs` to probably address all of the nits at once.